### PR TITLE
net-dialup/xl2tpd: version bump to 1.3.12 bug #656052

### PR DESCRIPTION
**Pull request by TeeDizzle**
Closes: https://bugs.gentoo.org/656052
Package-Manager: Portage-2.3.40, Repoman-2.3.9

### CODE REVIEW

File: net-dialup/xl2tpd/xl2tpd-1.3.12.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit systemd toolchain-funcs
+
+DESCRIPTION="A modern version of the Layer 2 Tunneling Protocol (L2TP) daemon"
+HOMEPAGE="http://www.xelerance.com/services/software/xl2tpd/"
```
Comment (mgorny) [inline net-dialup/xl2tpd/xl2tpd-1.3.12.ebuild]: Please include the verbatim GitHub homepage here, so we could easily copy-paste it into addressbar ;-).

File: net-dialup/xl2tpd/xl2tpd-1.3.12.ebuild [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+SRC_URI="https://github.com/xelerance/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
```
Comment (mgorny) [inline net-dialup/xl2tpd/xl2tpd-1.3.12.ebuild]: Interesting enough, https://github.com/xelerance/xl2tpd/blob/master/debian/copyright says GPL-2+.

File: net-dialup/xl2tpd/xl2tpd-1.3.12.ebuild (hunk 1 above)
Comment (TeeDizzle) [reply net-dialup/xl2tpd/xl2tpd-1.3.12.ebuild]: Good point, the formerly linked page is outdated anyway.

File: net-dialup/xl2tpd/xl2tpd-1.3.12.ebuild (hunk 2 above)
Comment (TeeDizzle) [reply net-dialup/xl2tpd/xl2tpd-1.3.12.ebuild]: Ah interesting indeed. They probably changed it at some point.

**Pull request closed without merging**